client/src/MergeAlerts.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Alert } from 'react-bootstrap'
+import { alertTable } from './constants'
+
+class Alerts extends React.Component {
+  render () {
+    return (
+      <div>
+        {displayAlert(this.props.alertType)}
+      </div>
+    )
+  }
+}
+
+function displayAlert (alertType) {
+  if (alertType === alertTable.MERGED) {
+    return (
+      <Alert variant='success'>
+      Account has been successfully merged.
+      </Alert>
+    )
+  } else if (alertType === alertTable.BAD_URL) {
+    return (
+      <Alert variant='danger'>
+      Invalid URL.
+      </Alert>
+    )
+  } else if (alertType === alertTable.NONEXISTENT_ACCOUNT) {
+    return (
+      <Alert variant='danger'>
+      This account does not exist.
+      </Alert>
+    )
+  } else if (alertType === alertTable.ALREADY_MERGED) {
+    return (
+      <Alert variant='danger'>
+      This account has already been merged. Please choose an unmerged account.
+      </Alert>
```

Comment:
Could you elaborate on this? Do you mean for empty inputs or something else? Im not quite following.

Reply:
function `displayAlert` returns different things based on the value of `alertType`. This comment points out that you do not return anything if `alertType` is unrecognized, an empty string, or undefined. It would be better if you handled all these cases to explicitly return null.

If you switched the syntax from if-else to switch-case, then the syntax would require you to handle those cases anyway.